scapy/layers/tls/cert.py

```diff
@@ -534,7 +534,7 @@ def import_from_asn1pkt(self, privkey):
         self.pubkey = self.key.public_key()
 
     @crypto_validator
-    def verify(self, msg, sig, h="sha256", **kwargs):
+    def verify(self, msg, sig, t="pkcs", h="sha256", mgf=None, L=None):
```

Comment:
Please undo these changes. Even if they trigger a warning in Codacy (which we'll ignore), they add unused zrguments

Reply:
This causes an exception for my example handshake with TLS_ECDHE_ECDSA_WITH_NULL_SHA, see https://ci.appveyor.com/project/secdev/scapy/builds/23842720/job/m09g9p06yrna1l4b
Please let me know if this could be fixed in different way.

Reply:
I will try to fix this without changing methods signature.

Reply:
I guess updating https://github.com/secdev/scapy/blob/cda583a17953964a08e21509103a193a0d759c48/scapy/layers/tls/cert.py#L304 and https://github.com/secdev/scapy/blob/cda583a17953964a08e21509103a193a0d759c48/scapy/layers/tls/cert.py#L513 to use keywords arguments will do